lib/Core/GPS/src/processUbloxGPS.c

```diff
@@ -499,15 +499,15 @@ void decodeNavPvt(char *msg, GpsData_t *GPSData)
 }
 
 void extractHeadingFromRelPosNed(char *msg, GpsData_t* GPSData) {
-    int relPosHeading = *( (int *) msg + 24 );
+    int32_t relPosHeading = *(int32_t*)(msg + 24);
```

Comment:
awf, I realize now that when you pointed that back in the office I looked the precedence table wrongly ... I saw unary operator +/- when I should have looked for addition operator. But even if it wasn't the case, I agree that better parentization would led to more readable and a lot less error prone code.

Reply:
Well. You know

IF YOU NEED TO READ A TABLE TO THINK THAT YOU KNOW PRECEDENCE, DON'T AND USE PARENTHESIS.